gettsim/taxes/eink_st.py

```diff
@@ -3,17 +3,17 @@
 from gettsim.typing import IntSeries
 
 
-def st_kein_kind_freib_tu(
-    zu_verst_eink_kein_kinderfreib_tu: FloatSeries,
```

Comment:
Let's please leave it at `freib` throughout, this goes way beyond the `kinderfreib`. Per GEP-01, would have to be `f_b` in the first place and there is no way to guess if you do not know the word.

Just make this private, seems perfectly fine to me to have it internal.

Below, everywhere where we had `kind_freib` -> `kinderfreib`